dist/index.html

```diff
@@ -12,29 +12,40 @@ <h1>Overlook</h1>
     <nav>
       <button>Book a room!</button>
       <button>Check your bookings!</button>
-      <button>Login</button>
     </nav>
   </header>
   <main>
-    <aside>
+    <section class="login">
+      <p>Login</p>
       <form>
-        <p>Filter</p>
-        <input type="date" name="Input date" />
-        <label for="room-type">Room Type
-          <select name="room-type" id="room-type">
-            <option value="none">None</option>
-            <option value="one-bed">One Bed</option>
-          </select>
-        </label>
-        <input type="submit" />
+        <label>Username</label>
+        <input type="text" required />
+        <label for="">Password</label>
+        <input type="password" required>
+        <input type="submit" value="Login">
       </form>
```

Comment:
I've been having a lot of trouble getting the required tags to work with forms and event.preventDefault(). Were you able to get both working together?